document/RuleMetadataProfile.mdk

```diff
@@ -2,13 +2,13 @@
 
 ## General
 
-It is sometimes useful for an issue management system to provide information to the end user
-about the rules that were violated by the issues in a log file.
-For example, the user might want to know whether the issue is a potential security risk,
-or where to find additional documentation for the issue.
+It is sometimes useful for a result management system to provide information to the end user
+about the rules that were evaluated in the course of producing the results in a log file.
+For example, the user might want to know whether the result indicates a potential security risk,
```

Comment:
suggest replacing 'the result indicates' with 'a problem indicates', because only problems will be evaluated for a security risk, and the change will break a bit of the repetitiveness around 'result'